hack/common.sh

```diff
@@ -56,8 +56,8 @@ MUST_GATHER_FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${MUST_GATH
 OCS_CLUSTER_UNINSTALL="${OCS_CLUSTER_UNINSTALL:-true}"
 OCS_SUBSCRIPTION_CHANNEL=${OCS_SUBSCRIPTION_CHANNEL:-alpha}
 
-UPGRADE_FROM_OCS_REGISTRY_IMAGE="${UPGRADE_FROM_OCS_REGISTRY_IMAGE:-quay.io/ocs-dev/ocs-registry:4.2.0}"
-UPGRADE_FROM_OCS_SUBSCRIPTION_CHANNEL="${UPGRADE_FROM_OCS_SUBSCRIPTION_CHANNEL:-$OCS_SUBSCRIPTION_CHANNEL}"
+UPGRADE_FROM_OCS_REGISTRY_IMAGE="${UPGRADE_FROM_OCS_REGISTRY_IMAGE:-$CATALOG_FULL_IMAGE_NAME}"
+UPGRADE_FROM_OCS_SUBSCRIPTION_CHANNEL="${UPGRADE_FROM_OCS_SUBSCRIPTION_CHANNEL:-4.4.0}"
```

Comment:
Does this mean, we'll start from `4.4.0` channel of latest `ocs-registry` ?
Will this upgrade to the latest image built by CI ?

Reply:
Since we are using hardcoded external image registry, this is still prone to error.
We should follow best practices given by OpenShift team.

But, this change will be bigger.

Reply:
Since we are using hardcoded external image registry, this is still prone to error.

Oops!

Reply:
The old image was also produced and mirrored by openshift CI automation, but it'll be much better using the same registry.